Backend/app/provider-platform/dynamic-offer-driver-app/Main/src/Domain/Action/UI/Driver.hs

```diff
@@ -1188,8 +1188,8 @@ respondQuote (driverId, _) req = do
       activeQuotes <- QDrQt.findActiveQuotesByDriverId driverId driverUnlockDelay
       logPretty DEBUG ("active quotes for driverId = " <> driverId.getId) activeQuotes
       pure $ not $ null activeQuotes
-    getQuoteLimit merchantId dist = do
-      driverPoolCfg <- DP.getDriverPoolConfig merchantId dist
+    getQuoteLimit merchantId dist searchTry = do
```

Comment:
here you can only pass vehicle varient no need to pass whole search try from above